drivers/include/cc2420.h

```diff
@@ -0,0 +1,313 @@
+/*
+ * Copyright (C) 2015-2016 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    drivers_cc2420 CC2420 driver
+ * @ingroup     drivers_netdev_netdev2
+ * @{
+ *
+ * @file
+ * @brief       Interface definition for the CC2420 driver
+ *
+ * @author      Thomas Eichinger <thomas.eichinger@fu-berlin.de>
+ * @author      Hauke Petersen <hauke.petersen@fu-berlin.de>
+ */
+
+#ifndef CC2420_H
+#define CC2420_H
+
+#include <stdint.h>
+
+#include "periph/spi.h"
+#include "periph/gpio.h"
+
+#include "net/netdev2.h"
+#include "net/netdev2/ieee802154.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief   Maximum possible packet size in byte
+ */
+#define CC2420_PKT_MAXLEN       (127U)
+
+/**
+ * @brief   Default addresses used if the CPUID module is not present
+ *
+ * In case this address is used, that short address will be created by using the
+ * last two bytes of the long address.
+ */
+#define CC2420_ADDR_FALLBACK    {0x12, 0x22, 0x33, 0x44, 0x55, 0x66, 0x08, 0x15}
+
+/**
+ * @brief   PAN ID configuration
+ */
+#define CC2420_PANID_DEFAULT    (0x0023)
+
+/**
+  * @brief   Channel configuration
+  */
+#define CC2420_CHAN_MIN         (11U)
+#define CC2420_CHAN_MAX         (26U)
+#define CC2420_CHAN_DEFAULT     (26U)
+
+/**
+ * @brief   Default TX power configuration [in dBm]
+ * @{
+ */
+#define CC2420_TXPOWER_MIN      (-25)
+#define CC2420_TXPOWER_MAX      (0)
+#define CC2420_TXPOWER_DEFAULT  (0)
+/** @} */
+
+/**
+ * @brief   A couple of return values used in this driver
+ */
+enum {
+    CC2420_RET_CHAN_OK      = 2,
+};
+
+/**
+ * @brief   Struct holding all parameters needed for device initialization
+ * @{
+ */
+typedef struct cc2420_params {
+    spi_t spi;              /**< SPI bus the device is connected to */
+    spi_speed_t spi_clk;    /**< SPI speed to use */
+    gpio_t pin_cs;          /**< pin connected to chip select */
+    gpio_t pin_fifo;        /**< pin connected to the FIFO interrupt pin */
+    gpio_t pin_fifop;       /**< pin connected to the FIFOP interrupt pin */
+    gpio_t pin_cca;         /**< pin connected to CCA */
+    gpio_t pin_sfd;         /**< pin connected to 'start of frame delimiter' */
+    gpio_t pin_vrefen;      /**< pin connected to the Vref enable pin */
+    gpio_t pin_reset;       /**< pin connected to the reset pin */
+} cc2420_params_t;
+/** @} */
+
+/**
+ * @brief   Device descriptor for CC2420 radio devices
+ * @{
+ */
+typedef struct {
+    /* netdev fields */
+    netdev2_ieee802154_t netdev;
```

Comment:
Yes me too and I fixed it but I'm also having this warnings:
```
warning: ignoring unknown tag `Copyright' at line 2, file cc2420.h
warning: ignoring unknown tag `C' at line 2, file cc2420.h
warning: ignoring unknown tag `2015' at line 2, file cc2420.h
warning: ignoring unknown tag `Freie' at line 2, file cc2420.h
warning: ignoring unknown tag `Universit' at line 2, file cc2420.h
warning: ignoring unknown tag `t' at line 2, file cc2420.h
warning: ignoring unknown tag `Berlin' at line 2, file cc2420.h
warning: ignoring unknown tag `This' at line 4, file cc2420.h
warning: ignoring unknown tag `file' at line 4, file cc2420.h
warning: ignoring unknown tag `is' at line 4, file cc2420.h
warning: ignoring unknown tag `subject' at line 4, file cc2420.h
warning: ignoring unknown tag `to' at line 4, file cc2420.h
```
and so on... I'll push it again and see if that was the warning Murdock is complaining on...